www-servers/automx2/automx2-2019.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_7 )
+
+inherit distutils-r1
```

Comment:
Looking at `init` file, perhaps you meant to use python-single-r1.eclass?

Reply:
I'll point to the [dev manual](https://devmanual.gentoo.org/eclass-reference/distutils-r1.eclass/index.html) again: "distutils-r1.eclass - A simple eclass to build Python packages using distutils."